packages/meditrak-app/ios/TupaiaMediTrak/Info.plist

```diff
@@ -17,11 +17,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.11.118</string>
+	<string>1.11.120</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>118</string>
+	<string>120</string>
```

Comment:
We got [a hotfix](https://beyondessential.slack.com/archives/CLVRJ3QBD/p1651706986536589) to master to bump version to 119. So this change would be 120.